tests/conftest.py

```diff
@@ -25,15 +25,28 @@ def system_tokens():
             tname + "/condarc",
             tname + "/condarc.d/",
         )
-        c_constants.SEARCH_PATH = n_path + o_path
+        c_constants.SEARCH_PATH = n_path
+        yield n_path
+        c_constants.SEARCH_PATH = o_path
+        utils._cache_clear("_search_path", "organization_token", "machine_token")
+
+
+@pytest.fixture
+def no_system_tokens():
```

Comment:
This new fixture makes sure conda cannot see any admin-installed tokens that might already be present. Necessary for testing locally now :-)